packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -279,9 +282,11 @@ export default class extends Component {
                 <tr>
                   <th>{_('hyperThreading')}</th>
                   <td>
-                    {this._isHtEnabled()
+                    {isHtEnabled === null
+                      ? _('hyperThreadingNotAvailable')
+                      : isHtEnabled === true
```

Comment:
```js 
isHtEnabled ? _('stateEnabled') : _('stateDisabled')
```
 ?

Reply:
the convention says: 
- if `isHtenabled` === `null` we display the message containing in this `_('hyperThreadingNotAvailable')`

Reply:
This proposition for the second test not for `isHtenabled === null`.